velox/functions/lib/aggregates/tests/utils/AggregationTestBase.cpp

```diff
@@ -959,6 +959,68 @@ void AggregationTestBase::testAggregationsImpl(
   }
 }
 
+void AggregationTestBase::testStreamingAggregationsImpl(
+    std::function<void(PlanBuilder&)> makeSource,
+    const std::vector<std::string>& groupingKeys,
+    const std::vector<std::string>& aggregates,
+    const std::vector<std::string>& postAggregationProjections,
+    std::function<std::shared_ptr<exec::Task>(AssertQueryBuilder&)>
+        assertResults,
+    const std::unordered_map<std::string, std::string>& config) {
+  {
+    SCOPED_TRACE("Run single");
```

Comment:
"Run single" may be confused with running hash single aggregation. Perhaps, add some indication that this is streaming agg.

Reply:
Fixed by adding "Test StreamingAggregation: run single".